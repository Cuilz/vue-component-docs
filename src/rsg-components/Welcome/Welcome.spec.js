import React from 'react';
import { WelcomeRenderer } from './WelcomeRenderer';

it('renderer should render welcome screen', () => {
	const actual = shallow(<WelcomeRenderer classes={{}} patterns={['foo/*.js', 'bar/*.js']} />);

	expect(actual).toMatchSnapshot();
});
