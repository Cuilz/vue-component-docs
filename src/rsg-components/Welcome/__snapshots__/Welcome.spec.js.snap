// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer should render welcome screen 1`] = `
<div>
  <Markdown
    text="
# Welcome to Vue Styleguidist!

**We couldnâ€™t find any components** using these patterns:

- \`foo/*.js\`
- \`bar/*.js\`

Create **styleguide.config.js** file in your project root directory like this:

    module.exports = {
      components: 'src/components/**/*.vue'
    };

Read more in the [locating components guide](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Components.md).
				"
  />
</div>
`;
